src/components/productListing/ProductCard.jsx

```diff
@@ -0,0 +1,49 @@
+import { FaRegHeart, FaHeart } from "react-icons/fa";
+import { useState } from "react";
+
+import { ProductHeader } from "./ProductHeader";
```

Comment:
make a single file to store all imports and use it everywhere. Looks clean.

Reply:
Noted. Thank you!